RoddyCore/src/de/dkfz/roddy/config/FilenamePattern.java

```diff
@@ -121,11 +119,13 @@ protected String fillVariables(BaseFile baseFile, String temp) {
     protected String fillVariablesFromSourceFileValues(BaseFile baseFile, String temp) {
         FileStageSettings fs = baseFile.getFileStage();
         String _temp = temp;
-        temp = fs.fillStringContent(temp);
+        if (fs != null)
```

Comment:
Well I totally know where it comes from. That is also, why I just catch it and ignore it. It can be something in the tests. It could be, that the class is not properly created. However, we can also discuss if we fail. But I'd leave that for a detailed discussion on error handling / catching and the strict mode, if it is ok for you.